projects/storefrontapp-e2e-cypress/cypress.json

```diff
@@ -2,15 +2,16 @@
   "defaultCommandTimeout": 10000,
   "baseUrl": "http://localhost:4200",
   "requestTimeout": 15000,
+  "chromeWebSecurity": false,
```

Comment:
Do we _really_ have to enable it? If yes, let's move it to `cypress.cpq.json`

Reply:
@schuppel , @schuppell Couldl you check?

Reply:
It is nevertheless possible to move to `cypress.cpq.json` but we should check why it was added in the first place.

Reply:
ok, we need it for our cypress ci landscape. So I will move it to `cypress.cpq.json`